CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.4.1 (2015-10-13)
```

Comment:
You should not set the release date in this commit. A dedicated commit updating CHANGELOG.md and README.md will have to be pushed to set this information ;)